reference-architecture/vmware-ansible/vars/infrastructure.yaml

```diff
@@ -0,0 +1,83 @@
+---
+infrastructure_hosts:
```

Comment:
Is there any way to tag or annotate hosts within vsphere that would allow introspecting these instead of hard-coding in a vars file?